Service/DRCService.php

```diff
@@ -299,12 +315,13 @@ private function generateDownloadEndpoint(string $id, Endpoint $downloadEndpoint
 
     }//end generateDownloadEndpoint()
 
-    public function createFilePart($object, $index, $size): array
+    public function createFilePart($object, $index, $size, string $lock): array
```

Comment:
I think you are missing a docblock here

Reply:
Hmm, that has been lost quite a while then, will add